client/k8scrdclient/k8scrdclient.go

```diff
@@ -89,3 +89,24 @@ func (c *CRDClient) Ensure(ctx context.Context, customResource *apiextensionsv1b
 
 	return nil
 }
+
+// EnsureDeletion ensures the given CRD does not exist.
+func (c *CRDClient) EnsureDeletion(ctx context.Context, customResource *apiextensionsv1beta1.CustomResourceDefinition, backOff backoff.BackOff) error {
+	operation := func() error {
+		err := c.k8sExtClient.ApiextensionsV1beta1().CustomResourceDefinitions().Delete(customResource.Name, nil)
+		if errors.IsAlreadyExists(err) {
```

Comment:
This doesn't follow for me - aren't we making sure it _doesn't_ exist?

Reply:
Copy/paste, should be `IsNotFound`.

Reply:
Done.